src/styles/blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid $grayElements;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    box-sizing: border-box;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+
+    //scale image and add border to it on hover
+    transition: transform 0.4s, width 0.4s;
+
+    &:hover {
+      transform: scale(1.2);
+      border: 3px solid $grayElements;
```

Comment:
don't add new border on hover, instead add one with transparent border, and it's color on hover, also mention it in transition